libs/sdk-backend-tiger/src/convertors/toBackend/afm/MeasureConverter.ts

```diff
@@ -102,7 +102,9 @@ function convertSimpleMeasureDefinition(definition: IMeasureDefinition): SimpleM
     const { measureDefinition } = definition;
 
     const filters: FilterDefinitionForSimpleMeasure[] = measureDefinition.filters
-        ? (compact(measureDefinition.filters.map(convertFilter)) as FilterDefinitionForSimpleMeasure[]) // measureDefinition.filters is IMeasureFilter, it contains only date and attribute filter, equally result contains this subset, it corresponds to type FilterDefinitionForSimpleMeasure
+        ? (compact(
+              measureDefinition.filters.map((filter) => convertFilter(filter)),
```

Comment:
Why this change?

Reply:
TS reports it as a problem otherwise:

<html>TS2345: Argument of type '(filter0: IAbsoluteDateFilter | { relativeDateFilter: { dataSet: ObjRef; granularity: DateAttributeGranularity; from: number; to: number; }; } | ... 5 more ... | IFilterWithApplyOnResult, afmAttributes?: IAttribute[]) =&gt; NegativeAttributeFilter | ... 7 more ... | null' is not assignable to parameter of type '(value: IMeasureFilter, index: number, array: IMeasureFilter[]) =&gt; NegativeAttributeFilter | PositiveAttributeFilter | ... 6 more ... | null'.<br/>Types of parameters 'afmAttributes' and 'index' are incompatible.<br/>Type 'number' is not assignable to type 'IAttribute[] | undefined'.

Reply:
that is because map function has signature: `(value: T, index: number, array: T[])`

Reply:
I see.